core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/SequenceActions.scala

```diff
@@ -175,7 +179,11 @@ protected[actions] trait SequenceActions {
               case Failure(t)   => logging.warn(this, s"activation event was not sent: $t")
             }
           }
-          activationStore.storeAfterCheck(seqActivation, context)(transid, notifier = None)
+
+          // Don't store the record when activation is successful, blocking, not in debug mode and no disable store is configured
+          if (!(seqActivation.response.isSuccess && blockingSequence && !transid.meta.extraLogging && disableSequenceStoreResultConfig)) {
```

Comment:
Is there a way to move this logic to a central location e.g. `shouldStoreActivation()`, so that it is more obvious what aspects are considered for this, and none are forgotten in some places where the logic is repeated?

Reply:
Yeah code duplication is not nice, will change, I was wondering were would be the best place to put this utils function

Reply:
@tysonnorris I've moved the logic for disabling the store in `ActivationStore`. 
There are already some namespace level configuration that disables the document store, I think it makes sense to have the system level checks defined in the same spot.